### PR TITLE
Splitting packaging-common into common-npm-packages

### PR DESCRIPTION
**Pull request by zhenghao104**
**Task name**: <Name of changed or new pipeline task>

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by egor-bryzgalov** [comment]
I'd like to suggest to create a `2.0.0-preview.0` version of this package at [common-packages-preview branch](https://github.com/microsoft/azure-pipelines-tasks/tree/common-packages-preview) dependent on `3.0.6-preview.0` version of `task-lib`.

**Comment by zhenghao104** [comment]
> I'd like to suggest to create a `2.0.0-preview.0` version of this package at [common-packages-preview branch](https://github.com/microsoft/azure-pipelines-tasks/tree/common-packages-preview) dependent on `3.0.6-preview.0` version of `task-lib` after these changes are merged.

Thank you for your comment. Do you mean doing the same thing (copying over the directory) on the common-packages-preview branch but set the version to preview version?

File: common-npm-packages/packaging-common/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "packaging-common",
```
Comment (egor-bryzgalov) [inline common-npm-packages/packaging-common/package.json:2]: I would propose to change it to `azure-pipelines-tasks-packaging-common`.

**Comment by egor-bryzgalov** [comment]
> > I'd like to suggest to create a `2.0.0-preview.0` version of this package at [common-packages-preview branch](https://github.com/microsoft/azure-pipelines-tasks/tree/common-packages-preview) dependent on `3.0.6-preview.0` version of `task-lib` after these changes are merged.
> 
> Thank you for your comment. Do you mean doing the same thing (copying over the directory) on the common-packages-preview branch but set the version to preview version?

I've opened a [PR for this](https://github.com/microsoft/azure-pipelines-tasks/pull/13978). Could you please take a look?

**Comment by egor-bryzgalov** [comment]
Could you please use `"azure-pipelines-tasks-utility-common": "^3.0.0-preview.0"` instead of the local one?

File: common-npm-packages/packaging-common/package.json (hunk 1 above)
Comment (zhenghao104) [reply common-npm-packages/packaging-common/package.json]: Updated the change.

File: common-npm-packages/packaging-common/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "version": "1.0.2",
+  "description": "Azure Pipelines Packaging Tasks Common",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "cd ../build-scripts && npm install && cd ../artifacts-common && node make.js"
```
[the remark this answers is not in the document]
Comment (zhenghao104) [reply common-npm-packages/packaging-common/package.json]: Fixed

**Review by egor-bryzgalov — APPROVED** [review]
LGTM. Thank you!

File: common-npm-packages/packaging-common/make.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "rm": [
```
Comment (damccorm) [inline common-npm-packages/packaging-common/make.json:2]: You don't need this file anymore

File: common-npm-packages/packaging-common/bump-task-version.ps1 [hunk 4]
```diff
@@ -0,0 +1,60 @@
+﻿# Bumps the version of all of the tasks that are technically modified when packaging-common is changed.
```
Comment (damccorm) [inline common-npm-packages/packaging-common/bump-task-version.ps1:1]: You don't need this file anymore

**Review by damccorm — APPROVED** [review]

File: common-npm-packages/packaging-common/make.json (hunk 3 above)
Comment (zhenghao104) [reply common-npm-packages/packaging-common/make.json]: Deleted

File: common-npm-packages/packaging-common/bump-task-version.ps1 (hunk 4 above)
Comment (zhenghao104) [reply common-npm-packages/packaging-common/bump-task-version.ps1]: Deleted

**Pull request merged**